app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,8 @@
+class PagesController < ApplicationController
+  include HighVoltage::StaticPage
+  before_action :authenticate_user!
+
+  def show
+    render 'pages/instructions'
+  end
```

Comment:
Not sure you need this method. I'm worried that you are locking yourself down to only one page. I know we don't have any feature requests for multiple static pages atm, but I would rather have the possibility to be flexible if a need for more pages should arise.